libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -1138,6 +1140,27 @@ bool AP_Arming::rc_checks_copter_sub(const bool display_failure, const RC_Channe
     return ret;
 }
 
+// check visual odometry is working
+bool AP_Arming::visodom_checks(bool display_failure) const
+{
+    if ((checks_to_perform != ARMING_CHECK_ALL) && !(checks_to_perform & ARMING_CHECK_VISION)) {
+        return true;
+    }
+
+#if HAL_VISUALODOM_ENABLED
+    AP_VisualOdom *visual_odom = AP::visualodom();
+    if (visual_odom != nullptr) {
+        char fail_msg[MAVLINK_MSG_STATUSTEXT_FIELD_TEXT_LEN+1];
+        if (!visual_odom->pre_arm_check(fail_msg, ARRAY_SIZE(fail_msg))) {
+            check_failed(ARMING_CHECK_VISION, display_failure, "%s", fail_msg);
```

Comment:
```suggestion
            check_failed(ARMING_CHECK_VISION, display_failure, "VisualOdom: %s", fail_msg);
```

... and remove `VisualOdom ` from all the places that include it in the prearm checks.  These bytes add up.

Reply:
ok thanks.  moved "VisualOdom:" to AP_Arming and removed from AP_VisualOdom lib.